new/LIT7072QeneMamhera.xml

```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7072QeneMamhera" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">">መምህረ፡ ዝንቱ፡ ዓለም፡ አውስጣቴዎስ፡ ደብረ፡ አርማንያ፡ ወልደከ፡ በጽሐ፡ ለዘምሮ፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Mamǝhǝra zǝntu ʿālam, ʾAwǝsṭātewos, dabra ʾArmānyā waldaka baṣḥa la-zammǝro… (Qǝne of unknown type)</title>
+                <title xml:lang="en" corresp="#t1">The teacher of this world, ʾEwosṭātewos, came to the mountain of Armenia to praise Your Son…</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="JK"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript. In the first line, it 
+                    refers to <persName ref="PRS3934Ewostate"/>.
+                    It is attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD"/> in the manuscript.</p>
```

Comment:
Could you please write out the name?

Reply:
Is this necessary? I thought it is normally enough to refer to the PRS record, or am I wrong?

Reply:
Yes, but for the sake of better readability, in a short text, it is word of doing. For instance when I look through it on GitHub, I have to go to the databank for this.

Reply:
Ok, I understand

Reply:
I am in favour of not writing as it minimizes errors. E. g. if at some point one would say we need to switch to Leslau/EAe transcription I only have to change the spelling once in PRS14574GabraLeul and not in an unlimited nnumber of records pointing to it through `ref`

Reply:
makes sense!